src/main/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrdersJpaRepository.java

```diff
@@ -16,7 +16,9 @@
 
     int deleteByCaseIdIsIn(Collection<Long> caseIds);
 
-    List<CasePaymentOrderEntity> findAllByCaseId(Long caseId);
+    int countAllById(UUID uuid);
+
+    int countAllByCaseId(Long caseId);
```

Comment:
may be exits query `boolean existsByCaseId(Long caseId)`

Reply:
👍